### DISCUSSION TITLE
adjust the position of the unit cube

### DISCUSSION THREAD

**Discussion by ggrp-china** [NVlabs/instant-ngp#593] [Q&A] [answered]
Hi, thanks for the great work!
I want to know is there any way to adjust the position of the cube,
I want to let this pavilion and cube same direction to ensure the integrity of the model when i adjust Crop aabb

![2022-05-04 02-52-53](https://user-images.githubusercontent.com/54228996/166522975-ed816099-f52d-4f1b-9419-d8b12fd885c7.png)

**Comment by Tom94** [comment, marked as answer]
Hi there, if you'd like to just translate and scale the unit cube, you can use the `scale` and `offset` parameters of `transforms.json` (see `nerf_loader.cu` for details on how they're used). If you also want to rotate, you will have to apply that to all the camera matrices.

**Reply by taylanoaydin** [reply]
Hi, do you have any tips on changing all the camera matrices such that the unit cube is rotated? Thanks.

**Reply by Tom94** [reply]
Take the 4x4 rotation matrix corresponding to your desired unit cube rotation and multiply all camera matrices with it.

**Reply by marco-lee25** [reply]
Hi, to rotate the unit cube, I wonder if each camera matrix has its own rotation matrix, orr just multiply them with the same one?
